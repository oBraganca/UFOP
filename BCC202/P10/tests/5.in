5
-53 82 -17 16 -1 -39 37 -21 36 15 -26 14 28 -23 2 27 -10 -20 33 23 21 -24 -14 13 -28 -16 19 -38 0 -9 -41 38 3 -25 11 29 -11 20 -36 40 -34 9 25 -5 -4 -31 -12 18 32 -15 -32 7 -27 8 -18 -29 -13 -2 -37 -40 35 34 4 6 22 -22 -30 -6 10 -8 -3 39 24 17 30 -7 5 -33 1 -19 26 -35 12 31
-16 86 -38 3 -34 -42 -14 28 26 25 33 -3 -43 -15 -33 -31 -36 -9 8 -8 20 -40 -25 36 27 17 -7 19 5 14 -29 -39 9 22 -22 12 7 -41 18 -12 30 -4 24 34 -1 39 -16 -37 -5 41 -23 21 -2 -13 1 6 29 -10 42 -19 -28 -18 23 4 16 10 37 -6 13 11 -30 -26 -17 -32 15 0 31 32 38 35 40 2 -11 -21 -24 -27 -20 -35
-6 73 16 25 -8 33 28 -27 -26 -13 12 -30 35 -24 -5 8 -4 -35 36 19 -12 17 -9 21 3 -14 -28 10 -33 20 9 13 -2 -32 32 -3 1 -1 22 -20 -16 -18 34 -36 14 -6 -23 -7 24 2 23 -31 -34 11 -10 -22 30 29 -29 7 18 -11 0 -21 26 -17 -19 15 6 31 4 -25 27 -15 5
14 34 16 -1 10 0 -7 -4 -9 5 1 3 -12 -10 2 -15 11 14 12 15 9 13 -6 -8 6 -2 -13 -11 -14 -3 4 8 -5 7 -16 -17
0 81 21 -35 -39 7 -3 -37 -32 19 32 -27 -11 2 11 24 -15 -14 -8 -9 -30 8 -7 -31 -33 -18 10 4 15 16 40 -24 -25 27 18 -5 35 28 31 29 23 38 -20 22 -22 39 -10 0 1 5 -1 -29 -12 -38 9 -2 -36 33 14 6 25 12 -6 34 37 -21 36 -16 -13 17 13 -34 26 -17 -19 3 20 30 -23 -26 -28 -40 -4
