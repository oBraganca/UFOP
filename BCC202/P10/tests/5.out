82 6
45 7
44 7
16 4
46 1
